apps/cowswap-frontend/src/modules/limitOrders/containers/LimitOrdersWidget/index.tsx

```diff
@@ -216,7 +220,13 @@ const LimitOrders = React.memo((props: LimitOrdersProps) => {
         </>
       )
     },
-    outerContent: <>{isUnlocked && <InfoBanner />}</>,
+    outerContent: (
```

Comment:
I think you can also handle this logic inside of `TradeWidget` component like you have done with `topContent` and `lockScreen`